xbmc/music/MusicDatabase.cpp

```diff
@@ -12050,7 +12050,7 @@ void CMusicDatabase::ExportToXML(const CLibExportSettings& settings,
             {
               // Save art in album folder
               // Note thumb resolution may be lower than original when overwriting
-              std::map<std::string, std::string> artwork;
+              KODI::ART::ArtWork artwork;
```

Comment:
In cpp files, you can add a `using namespace KODI;` to the top, and shorten art stuff in the files to just `ART::`.

Moving everything to a KODI namespace could be a grunt task taken on by some future hero. In that case it would be more work removing `KODI::` everywhere, and more work adding `using namespace KODI;` which could already be added here.

But just a suggestion, not required for this PR.